### PR TITLE
Prevent to have utf-8 characters in attachment filenames #4099

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue #4099 .

### CODE REVIEW

File: src/administrator/components/com_kunena/language/en-GB/en-GB.com_kunena.views.ini [hunk 1]
```diff
@@ -535,6 +535,8 @@ COM_KUNENA_CONFIG_ATTACHMENT_SHORTHEN_NAME_END = "Number of characters from end
 COM_KUNENA_CONFIG_ATTACHMENT_SHORTHEN_NAME_END_DESC = "Define the number of characters from end when shorthen attachments filemane"
 COM_KUNENA_CONFIGURATION_GOOGLE_MAPS_API_KEY = "Google maps API key to get better stats usage of google API"
 COM_KUNENA_CONFIGURATION_GOOGLE_MAPS_API_KEY_DESC  = "Google maps API key to get better stats usage of google API. You need to enable the Google Maps Geocoding API in the developer console. <a href='https://console.developers.google.com/flows/enableapi?apiid=maps_backend&keyType=CLIENT_SIDE&reusekey=true' target='_blank'>Ask your API key</a>"
+COM_KUNENA_CONFIGURATION_ATTACHMENT_FILENAME_UTF8 = "Remove utf8 characters in attachments filemane"
```
Comment (810) [inline src/administrator/components/com_kunena/language/en-GB/en-GB.com_kunena.views.ini]: typo filename

**Comment by 810** [comment]
I think the option need to be reverted.

attachment_utf8 = Enabled (default) = example 你好.jpg

**Comment by xillibit** [comment]
Updated

File: src/libraries/kunena/config.php [hunk 2]
```diff
@@ -1159,6 +1159,12 @@ class KunenaConfig extends JObject
 	public $google_map_api_key = '';
 	
 	/**
+	 * @var integer Allow to remove utf8 characters from filename of attachments
+	 * @since  K5.0.0
+	 */
+	public $attachment_utf8 = 0;
```
Comment (810) [inline src/libraries/kunena/config.php]: change to 1

File: src/administrator/components/com_kunena/language/en-GB/en-GB.com_kunena.views.ini (hunk 1 above)
Comment (810) [inline src/administrator/components/com_kunena/language/en-GB/en-GB.com_kunena.views.ini]: change text to:
Enable utf-8 characters on attachments

**Pull request merged**